index.js

```diff
@@ -65,6 +65,7 @@ const config = {
             },
         ],
         curly: ["error"],
+        "functional/prefer-immutable-types": "off",
```

Comment:
It's not clear, but this is actually a generated file. `index.ts` is the one to update. I forget why we were forced (?) to commit the generated js. It certainly needs to part of the bundle that becomes published npm package but maybe we can remove from git to reduce confusion.

Reply:
I've updated `.ts` instead and rebuild the `.js` files. How is this package published?